test/api.test.ts

```diff
@@ -28,14 +28,14 @@ suite(`AzureFunctionsExtensionApi`, () => {
         const projectSubpath = 'api';
         const folderPath: string = path.join(workspaceFolder, projectSubpath);
 
-        await runWithInputs('api.createFunction', [language, /Model V3/, functionName], registerOnActionStartHandler, async () => {
+        await runWithInputs('api.createFunction', [language, NodeModelInput[NodeModelVersion.v3], functionName], registerOnActionStartHandler, async () => {
             await api.createFunction({
                 folderPath,
                 suppressOpenFolder: true,
                 templateId: 'HttpTrigger',
                 languageFilter: /Python|C\#|^(Java|Type)Script$/i,
                 functionSettings: { authLevel: 'anonymous' },
-                targetFramework: ['netcoreapp3.1', 'net6.0'] // Will only work on functions api v1.4.0, but won't hurt on v1.3.0
+                targetFramework: ['netcoreapp3.1', 'net6.0', 'net8.0'] // Will only work on functions api v1.4.0, but won't hurt on v1.3.0
```

Comment:
I think to test what we really have, it should be ['net6.0', 'net7.0', 'net8.0']

Reply:
Done